jaas/src/main/java/com/griddynamics/jagger/config/DataServiceConfig.java

```diff
@@ -14,21 +17,32 @@
 /**
  * Spring config for {@link com.griddynamics.jagger.engine.e1.services.DataService}.
  */
-@PropertySources({@PropertySource("classpath:/basic/default.environment.properties")})
-@ImportResource({"classpath:/common/hibernate.conf.xml", "classpath:/spring/dbapi.config.xml"})
+@PropertySources({@PropertySource("classpath:/basic/default.environment.properties"),
+                  @PropertySource("classpath:/reporter/default.reporting.properties"),
+                  @PropertySource("classpath:/jagger.properties"),
+                  @PropertySource(value = "${jagger.reporting.properties:''}", ignoreResourceNotFound = true)
```

Comment:
It is a pattern which allows to override properties at deployment time. You creates a file with properties you wanna override and pass its path to java via -Djagger.reporting.properties=<path to properties>